ibis/tests/backends.py

```diff
@@ -127,7 +128,16 @@ def connect(self, data_directory):
             'functional_alltypes': pd.read_csv(
                 str(data_directory / 'functional_alltypes.csv'),
                 index_col=None,
-                dtype={'bool_col': bool, 'string_col': six.text_type},
+                dtype={
+                    'id': np.int32,
+                    'bool_col': bool,
```

Comment:
Let's use `np.bool_` to be consistent with the rest of the dtypes specified here.